DrainageSystem-1/Readme.md

```diff
@@ -55,10 +55,11 @@ Network topology using `IfcDistributionPort` and `IfcRelConnectsPorts`.
 Following files correspond to this scenario:
 
 | Filename                 | Description                                                  |
-|:------------------------ | ------------------------------------------------------------ |
+| :----------------------- | ------------------------------------------------------------ |
 | `DrainageSystem-1-1.xml` | Inframodel/LandXML: PipeNetwork with Structs and Pipes       |
 | `DrainageSystem-1-2.xml` | Inframodel/LandXML: Top surface of the road and slopes/ditches |
 | `DrainageSystem-1-3.jpg` | Inframodel/LandXML: Screenshot plan view                     |
 | `DrainageSystem-1-4.jpg` | Inframodel/LandXML: Screenshot 3d view                       |
+| DrainageSystem-1-1.ifc   | IFC file implementing the drainage system according to the above content description<br />except that the IfcRoad was not decomposed into IfcFacilityPart.<br />The physical objects have both an Axis and a Body representation. |
```

Comment:
Please don't use html tags in text. Let the UI break the lines for you.

Reply:
Should be fixed.

Reply:
So, as you see, it doesn't always look so good when letting the UI do the breaks...